Fix GCC 14 compilation (Re-enable disabled int dbp_config_init() in config.h
Fix debian/control for trixie (dh-sequence-systemd instead of dh-systemd)
Index: dbp/debian/control
===================================================================
--- dbp.orig/debian/control
+++ dbp/debian/control
@@ -2,7 +2,7 @@ Source: dbp
 Section: misc
 Priority: optional
 Maintainer: Pyra Dev <pyra-dev@pyra-handheld.org>
-Build-Depends: debhelper (>= 9), dh-systemd, libarchive-dev, libdbus-1-dev, valac, libdpkg-dev, libimlib2-dev, libgtk-3-dev
+Build-Depends: debhelper (>= 9), dh-sequence-systemd, libarchive-dev, libdbus-1-dev, valac, libdpkg-dev, libimlib2-dev, libgtk-3-dev
 Standards-Version: 3.9.5
 Homepage: https://github.com/slaeshjag/dbp
 
Index: dbp/libdbpbase/config.h
===================================================================
--- dbp.orig/libdbpbase/config.h
+++ dbp/libdbpbase/config.h
@@ -44,7 +44,7 @@ struct DBPConfig {
 };
 
 
-//int dbp_config_init();
+int dbp_config_init();
 extern struct DBPConfig dbp_config_struct;
 char *dbp_config_version_get();
 void dbp_config_expand_token(char ***target, int *targets, char *token);
